css/main.css

```diff
@@ -1,3 +1,222 @@
-/* Add your CSS here */
+/* 
+Fonts
+Headings: Poppins - Semi-bold 600
+https://fonts.google.com/specimen/Poppins
+Body: Quicksand - Light 300
+https://fonts.google.com/specimen/Quicksand
+
+Colors
+Dark beige: #E6DACD
+Light beige: #F4ECE6
+Royal blue: #0150FD
+Black: #000
+White: #FFF */
 
+/* Add your CSS here */
 /* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --dark-beige: #e6dacd;
+  --light-beige: #f4ece6;
+  --royal-blue: #0150fd;
+  --black: #000;
+  --white: #fff;
+}
+
+body {
+  width: 320px;
```

Comment:
It is best never to set the width on the body, in this case as we are doing mobile first, you don't have to set a width at all, just adding one for the width with the media query for bigger screens will be sufficient